documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,444 @@
++++
+title = "Common Mounts"
+date = 2019-02-23T16:45:16-05:00
+weight = 70
+pre = "<b> </b>"
++++
+
+#### Contents
+
+ - [Introduction](#introduction)
+ - [Configuration](#configuration)
+ - [Common Mounts Fields](#common-mounts-fields)
+ - [Common Mount Volumes Fields](#common-mount-volumes-fields)
+ - [Running Model in Image initial use case using common mounts](#running-model-in-image-initial-use-case-using-common-mounts)
+    - [Creating the common mounts image](#creating-the-common-mounts-image)
+    - [Domain resource](#domain-resource)
+
+### Introduction
+Use common mounts to automatically include the directory content from additional images. This is a useful alternative for including Model in Image model files, or other types of files, in a Pod without requiring modifications to the Pod's base image specified using `domain.spec.image`. This feature internally uses a Kubernetes [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volume and Kubernetes [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) to share the files from the additional images with the Pod.
+
+When using the common mounts feature with the Model In Image use case, the WebLogic image would be the Domain resource's `domain.spec.image` field. The model files, application archives, and the WDT installation files can be provided in a separate image specified as part of the common mounts. The application archive could also be provided by a second container image in the common mounts section. Specific advantages of the common mounts feature for Model In Image domains are:
+  - Use a WebLogic install image, or patch a WebLogic install image, without needing to include model artifacts within the image.
+  - Share one large WebLogic install image with multiple different model configurations.
+  - Distribute model files, application archives, and WebLogic Deploy Tooling executable using very small images instead of a large image that also contains a WebLogic install.
+
+### Configuration
+Here's an example configuration for the Common Mounts. 
```

Comment:
Accounting for feature gate:

Until this feature is officially released, the feature gate usage for the feature needs to mentioned in here and in the sample section below.  

It also needs to be integrated into the sample test: in the test script that deploys the operator when '-oper' is specified for 'run-test.sh' stand-alone (which in turn calls `deploy-operator.sh` which needs to modified to enable the feature gate), and in the java integration test itself when you update it to support common mounts.  (If the feature gate code is working, it should have been impossible for 'run-test.sh -oper -cm ...' stand-alone to pass without this modification.)